File: server-mh/src/Match/Handler/Models/Cell.ts
export class Cell {
    public index:number;
    public isInitial: boolean;
    public isSafe: boolean;
    public isPenalty: boolean;
    public isFinal: boolean;

    constructor(
        index:number,
        isInitial: boolean = false,
        isSafe: boolean = false,
        isPenalty: boolean = false,
        isFinal: boolean = false
    ) {
        this.index=index;
        this.isInitial = isInitial;
        this.isSafe = isSafe;
        this.isPenalty = isPenalty;
        this.isFinal = isFinal;
    }
}

File: Client-unity/Ludo/Assets/Scripts/OfflineMatchHandler/Cell.cs
namespace Ludo.Offline
{
    public class Cell
    {
        public int Index { get; }

        public bool IsInitial { get; set; }
        public bool IsSafe { get; set; }
        public bool IsPenalty { get; set; }
        public bool IsFinal { get; set; }

        public Cell(int index)
        {
            Index = index;
        }
    }
}